packages/extension-base/src/defaults.ts

```diff
@@ -2,9 +2,12 @@
 // SPDX-License-Identifier: Apache-2.0
 
 const ALLOWED_PATH = ['/', '/account/import-ledger', '/account/restore-json'] as const;
-const PORT_CONTENT = 'content';
 const PHISHING_PAGE_REDIRECT = '/phishing-page-detected';
-const PORT_EXTENSION = 'extension';
+const EXTENSION_PREFIX = process.env.EXTENSION_PREFIX as string || '';
```

Comment:
```suggestion
const EXTENSION_PREFIX = typeof process === 'object' && typeof process.env === 'object' ? (process.env.EXTENSION_PREFIX as string || '') : '';
```

Small suggestion, had some weird stuff with environments breaking in the past - however, this also should be optional, at this level if/when it does break the users should know how to fix it. (Since the target here is extension devs)